string-manipulation-2/src/capitalize-word.js

```diff
@@ -10,10 +10,10 @@
 
 function capitalizeWord(word) {
   var output = '';
-  output = word[0].toUpperCase(word[0]);
+  output = word[0].toUpperCase();
   for (var i = 1; i < word.length; i++) {
-    if (word.toUpperCase(word) !== 'JAVASCRIPT') {
-      output += word[i].toLowerCase(word[i]);
+    if (word.toUpperCase() !== 'JAVASCRIPT') {
+      output += word[i].toLowerCase();
```

Comment:
You should check if the word is "JavaScript" before the loop so that this loop only runs if the word isn't JavaScript. That should remove the need for a conditional in your loop 🤓

Reply:
Just saw that you mentioned that earlier! Will be sure to have everything fixed before resubmitting!